content/positions/data-infrastructure-engineering-intern/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Data Infrastructure Engineering Intern (UC Student)"
+summary: 
+
+date:
+show_date: false
+profile: false
+image:
+  focal_point: ''
+  preview_only: true
+
+draft: false
+---
+
+
+## Position summary
+The FOCAL Lab at UC Davis is recruiting an undergraduate Data Infrastructure Engineering Intern to support the lab’s work at the intersection of forest ecology, forest mapping, and forest management. The intern will learn about and contribute to the implementation of open source infrastructure tools and frameworks to automate large-scale processing and cataloging of drone imagery and other types of data used to answer ecological questions. Specifically, the intern will be responsible for (a) advancing the adoption of [Argo Workflows](https://argoproj.github.io/workflows/) for orchestrating highly parallel, multi-step drone imagery processing workflows on cloud-based Kubernetes clusters, (b) improving automated tracking and cataloging of metadata, including transitioning from files to a relational PostgreSQL database, (c) automated archiving of data files in the [CyVerse Data Store](https://cyverse.org/data-store), and (d) if time, interest, and skills allow, developing a public web portal for browsing and accessing the processed and cataloged data and, ultimately, uploading new data.
```

Comment:
First sentence: this talks about the ecology domains we work in. But I think including some of the technical considerations is useful too, such as big data, cloud infrastructure, or open source software development.